client/my-sites/site-settings/press-this/index.jsx

```diff
@@ -50,13 +50,17 @@ class PressThis extends Component {
 						) }
 					</p>
 					<p className="pressthis">
```

Comment:
Maybe it would make sense to not display the `<p>` if the site is `null` - currently, if there's no `site`, an empty `<p>` will be displayed.